participants/verity-brean/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+        <html lang="en">
+        <head>
+          <meta charset="UTF-8">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <meta http-equiv="X-UA-Compatible" content="ie=edge">
+          <link rel="stylesheet" href="styles.css">
+          <title>Verity Brean</title>
+        </head>
+        <body>
+            <div class = "container">
+            <h1>Happy Halloween!</h1>
+            <img src = "ghost.png">
```

Comment:
might want to format this! :)

Reply:
Just formatted and pushed another commit! :)